src/validators/BaseValidator.ts

```diff
@@ -67,7 +68,7 @@ export abstract class BaseValidator<T> {
 		return Reflect.construct(this.constructor, [this.constraints]);
 	}
 
-	protected abstract handle(value: unknown): Result<T, ValidationError | CombinedError | CombinedPropertyError>;
+	protected abstract handle(value: unknown): Result<T, ValidationError | CombinedError | CombinedPropertyError | UnknownEnumValueError>;
```

Comment:
At this point, we should make this a type so we don't have to update all base validators.